workflows/spaceranger/spaceranger_count.wdl

```diff
@@ -232,8 +232,15 @@ task run_spaceranger_count {
             return darkimages
 
         has_cyta = not_null('~{cytaimage}')
+        probe_set = "~{probe_set}"
+        dapi_idx = "~{dapi_index}"
         if not_null('~{probe_file}'):
-            call_args.extend(['--probe-set=~{probe_file}', '--filter-probes=~{filter_probes}'])
+            call_args.append('--probe-set=~{probe_file}')
+            if version.parse('~{spaceranger_version}') >= version.parse('2.0.0'):
+                call_args.append('--filter-probes=~{filter_probes}')
+            else:
+                if '~{filter_probes}' == 'false':
+                    call_args.append('--no-probe-filter')
             if has_cyta and probe_set == "human_probe_v1":
```

Comment:
This can be
```python
if has_cyta and "~{probe_set}" == "human_probe_v1":
```

Reply:
Prone to side effects, I will put it like this, but it might randomly fail, as it was before